docs/concepts/ClusterResourcePlacement/README.md

```diff
@@ -0,0 +1,222 @@
+# ClusterResourcePlacement
+
+## Overview
+
+`ClusterResourcePlacement` concept is used to dynamically select cluster scoped resources (especially namespaces and all 
+objects within it) and control how they are propagated to all or a subset of the member clusters.
+A `ClusterResourcePlacement` mainly consists of three parts:
+- **Resource selection**: select which cluster-scoped Kubernetes
+resource objects need to be propagated from the hub cluster to selected member clusters. 
+  
+  It supports the following forms of resource selection:

```

Comment:
We could possibly include examples of each of these for users to reference back to. Maybe like a diagram to show what resources would be applied in each instance.

Reply:
would like to create a separate "How-to" README for each case. will add a sample yaml file there.